mlfinlab/data_structures/run_data_structures.py

```diff
@@ -197,11 +197,10 @@ def _get_expected_imbalance(self, window, imbalance_array):
                 imbalance_array['buy'][-ewma_window:], dtype=float)
             sell_sample = np.array(
                 imbalance_array['sell'][-ewma_window:], dtype=float)
-            buy_and_sell_imb = sum(buy_sample) + sum(sell_sample)
```

Comment:
@proskurin have you seen these changes since your last approval?

Reply:
Sorry, didn't see that. @atokuz for proportion we need to divide by buy_and_sell_imb, by simply taking buy_sample we don't get proportion